geode-redis/src/main/java/org/apache/geode/redis/internal/RedisConstants.java

```diff
@@ -30,6 +30,8 @@
   public static final String SERVER_ERROR_MESSAGE =
       "The server had an internal error please try again";
   public static final String SERVER_ERROR_SHUTDOWN = "The server is shutting down";
+  public static final String ERROR_SELECT_CLUSTER_MODE = "SELECT is not allowed in cluster mode";
```

Comment:
I'm afraid that this error will be a bit misleading since it mentions `cluster mode`. I think the `invalid DB index` is better and we should document the limitation of only supporting DB 0. I do realize that this is a bit inconsistent with how native redis reports things.

Reply:
I definitely understand what you mean.  I think it would be good for us to talk about this with the team.  There is also an `ERR DB index is out of range` message Redis returns if the user tries to select a DB that is not 0-15, but is still a long.  We could potentially return that instead.

Reply:
After some discussion, we decided to return `ERR Only DB 0 supported` if anything other than 0 is passed in as a DB index argument.